subsys/power/device.c

```diff
@@ -27,8 +27,8 @@ LOG_MODULE_DECLARE(power);
 #define NUM_CORE_DEVICES	4
 #define MAX_DEV_NAME_LEN	16
 static const char core_devices[NUM_CORE_DEVICES][MAX_DEV_NAME_LEN] = {
-	"clk_k32src",
-	"clk_m16src",
+	"CLOCK_0_32K",
+	"CLOCK_0_16M",
```

Comment:
After renaming `"CLOCK_0"` to `"CLOCK"` these two entries should be updated, too.

Reply:
Yes, Thank you for pointing that.

Reply:
Fixed